src/UI/Seller/src/styles/oc-theme/_oc-bootstrap-variables.scss

```diff
@@ -62,3 +53,8 @@ $theme-colors: (
 
 $body-bg:           white;
 $body-color:          $gray-800;
+
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300;0,400;0,600;0,700;0,800;1,400&display=swap');
+
+$font-family-sans-serif:      -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", "Liberation Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji" !default;
```

Comment:
might be just for my knowledge, but is this a part of the sitecore style guide? or where's the apple specific code coming from?

Reply:
there are tons of modern font stacks that are recommended nowadays, this is the stack [Bootstrap](https://getbootstrap.com/docs/5.0/utilities/text/#variables) uses.
Better answer here: https://css-tricks.com/snippets/css/system-font-stack/

Reply:
I guess the crux of my question is, if we are changing the fonts and style to Sitecore, we should be using Sitecore's styleguide for fonts also. https://design.sitecore.net/document/100045#/visual-design/typography 

Just making sure this stack is the one recommended from Sitecore, i'm seeing the open sans wght 300 that they use, but that's the only familiar one i'm seeing.

Reply:
for sure for sure. so line 60, that's exactly what I'm doing: 
`$font-family-base: "Open Sans", $font-family-sans-serif;`. 

However, because Boostrap doesn't know what the variable `$font-family-sans-serif` means yet, I need to jump over to node_modules, and grab the default definition, and define it in my custom variables file in order to change the "base" font stack. That's fallback is default Bootstrap.

Reply:
got it, sounds good. I figured it was just a lack of knowledge of my part. thanks!